docs/json.md

```diff
@@ -155,6 +155,13 @@ The type specifies the underlying data type and links to the constant via the fo
 ```
 The names of the components of the constant are written as external (JSON) values to the JSON Schema after being transformed to camel case (e.g, component `badi_definition` is transformed to the enum value `badiDefinition` in the JSON Schema).
 The corresponding values of the components represent the internal (ABAP) values.
+
+If not the transformed name of a component shall be used, the enum value can be overwritten by adding an ABAP Doc comment with the annotational keyword
+```abap
+"! $enumValue 'newEnumValue'
+```
+to the specific component.
+
```

Comment:
Does it make sense to mention that we recommend to use this addition rarely?

Reply:
I think this is a good idea. See my next commit